README.md

```diff
@@ -25,25 +35,80 @@ jobs:
     name: Add issue to project
     runs-on: ubuntu-latest
     steps:
-      # Pointing to a branch name generally isn't the safest way to refer to an action,
-      # but this is how you can use this action now before we've begun creating releases.
-      # Another option would be to point to a full commit SHA.
       - uses: actions/add-to-project@main
         with:
           project-url: https://github.com/orgs/<orgName>/projects/<projectNumber>
           github-token: ${{ secrets.ADD_TO_PROJECT_PAT }}
           labeled: bug
 ```
 
+#### Further reading and additional resources
+
+- [Inputs](#inputs)
+- [Supported Events](#supported-events)
+- [How to point the action to a specific branch or commit sha](#how-to-point-the-action-to-a-specific-branch-or-commit-sha)
+- [Creating a PAT and adding it to your repository](creating-a-pat-and-adding-it-to-your-repository)
+- [Development](#development)
+- [Publish to a distribution branch](#publish-to-a-distribution-branch)
+
 ## Inputs
 
-- `project-url` is the URL of the GitHub Project to add issues to.
-- `github-token` is a [personal access
+- <a name="project-url">`project-url`</a> **(required)** is the URL of the GitHub Project to add issues to.  
+  _eg: `https://github.com/orgs|users/<ownerName>/projects/<projectNumber>`_
```

Comment:
I added this here for now, because the `project-url` currently has to be on `github.com`, meaning that we don't support enterprise URLs.

There's a todo in the code:
```
// TODO: Ensure this (and the Octokit client) works for non-github.com URLs, as well.
// https://github.com/orgs|users/<ownerName>/projects/<projectNumber>
```

which I've added to the tracking issue to make sure we hunt down whether there is octokit support for enterprise urls.

Reply:
that's a good call out, but we should be ok to not worry about this yet either, since beta projects are not on enterprise yet